config/lib/mongoose.js

```diff
@@ -43,3 +43,34 @@ module.exports.disconnect = function (cb) {
     cb(err);
   });
 };
+
+module.exports.dropDatabase = function(cb) {
```

Comment:
Mind renaming `cb` to `callBack`, so that it is more explicit what the parameter is?

Reply:
Im cool with either.  It seems like the census online is to use "cb" but I generally prefer being verbose